src/sentry/api/endpoints/organization_details.py

```diff
@@ -168,6 +168,8 @@ class OrganizationSerializer(BaseOrganizationSerializer):
     allowJoinRequests = serializers.BooleanField(required=False)
     relayPiiConfig = serializers.CharField(required=False, allow_blank=True, allow_null=True)
     apdexThreshold = serializers.IntegerField(min_value=1, required=False)
+    authProvider = serializers.CharField(required=False, allow_blank=True, allow_null=True)
+    providerConfig = serializers.JSONField(required=False, allow_null=True)
```

Comment:
@schew2381 Can you add a validator for this field? Right now, it seems users can pass in arbitrary JSON payloads.

Reply:
Is there somewhere in the code where I can see an example config for authProvider? I looked in the `AuthProvider` class variables/functions and tried to find instances using config but I couldn't find anything useful

https://github.com/getsentry/sentry/blob/master/src/sentry/models/authprovider.py#L45

Reply:
After doing some digging it looks like the config gets validated upon trying to log in, so instead of validating it here we can simply call `build_config()` to build the appropriate config based on the provider and let it validate later